server.js

```diff
@@ -347,7 +347,7 @@ async function generateDevelopmentBuild() {
   });
 
   //eslint-disable-next-line no-unused-vars
-  app.get("/proxy{/:remote}", function (req, res, next) {
+  app.get("/proxy{/*remote}", function (req, res, next) {
```

Comment:
I don't think this will be picked up as a parameter without `:`. What am I missing?

Reply:
Using the `:` will not allow multiple nested routes. This is what breaks this version of the proxy url: http://localhost:8080/proxy/http://example.com/file?query=1

I don't know why it's not documented well in the [route params](https://expressjs.com/en/guide/routing.html) docs, maybe another issue to open on their side. Doing it this way is how they say to replace wildcard routes in the [Migrating to v5 page](https://expressjs.com/en/guide/migrating-5.html#path-syntax)

Reply:
So does the wildcard still triggers the `app.param` callback?